### PR TITLE
Update dlp-conditions-and-exceptions.md

### PR DESCRIPTION
**Pull request by ybhargav1995**

### CODE REVIEW

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ To configure the sender address location at a DLP rule level, the parameter is *
 
 |condition or exception in DLP|condition/exception parameters in Security & Compliance PowerShell|property type|description|
 |---|---|---|---|
-|Attachment is password protected|condition: *DocumentIsPasswordProtected* <br/><br/> exception: *ExceptIfDocumentIsPasswordProtected*|none|Messages where an attachment is password protected (and therefore can't be scanned). Password detection only works for Office documents, .zip files, and .7z files.|
+|Attachment is password protected|condition: *DocumentIsPasswordProtected* <br/><br/> exception: *ExceptIfDocumentIsPasswordProtected*|none|Messages where an attachment is password protected (and therefore can't be scanned). Password detection works for Office documents, archive documents (.zip, .7z, .rar, .tar etc.) and .pdf files.|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
|Attachment is password protected|condition: *DocumentIsPasswordProtected* <br/><br/> exception: *ExceptIfDocumentIsPasswordProtected*|none|Messages where an attachment is password protected (and therefore can't be scanned). Password detection works for Office documents, archive documents (such as .zip, .7z, .rar, and .tar files), and .pdf files.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ybhargav1995, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ybhargav1995** [comment]
@chrfox - Can this be merged?

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**